### PR TITLE
Sort plugins on debugpaste

### LINKED ISSUE #1428: Sort plugins in debugpaste alphabetically
### What feature do you want to see added?

For better visibility and readability, we could order plugins alphabetically by their names, e.g. how they appear in timings or on `/plugins` when you're using Paper.

### Are there any alternatives?

No

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by Owen1212055**
## Overview

Resolves #1428

## Description

Replace plugins with a list that is sorted using the compareToIgnoreCase statement. (The plugins command uses the CASE_INSENSITIVE_ORDER comparator, which this method uses this same comparator) 

## Checklist

- [X] Make sure you are opening from a topic branch (**/feature/fix/docs/ branch** (right side)) and not your main branch.
- [X] Ensure that the pull request title represents the desired changelog entry.
- [X] I tested my changes and approved their functionality.
- [X] I ensured my changes do not break other parts of the code
- [X] I read and followed the [contribution guidelines](https://github.com/IntellectualSites/.github/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Review by dordsor21 — APPROVED** [review]
Fair enough aha

**Review by NotMyFault — APPROVED** [review]
LGTM, thanks for the contribution!

We may merge it in 24 hours if there is no negative feedback, as per the [merge process ](https://github.com/IntellectualSites/IntellectualSites/blob/main/docs/MAINTAINERS_CONTRIBUTORS.adoc#merge-process) documentation.

File: worldedit-bukkit/src/main/java/com/fastasyncworldedit/bukkit/FaweBukkit.java [hunk 1]
```diff
@@ -148,9 +150,13 @@ public ItemUtil getItemUtil() {
     @Override
     public String getDebugInfo() {
         StringBuilder msg = new StringBuilder();
-        Plugin[] plugins = Bukkit.getServer().getPluginManager().getPlugins();
+
+        List<Plugin> plugins = new ArrayList<>();
+        Collections.addAll(plugins, Bukkit.getServer().getPluginManager().getPlugins());
+        plugins.sort((plugin1, plugin2) -> plugin1.getName().compareToIgnoreCase(plugin2.getName()));
```
Comment (SirYwell) [inline worldedit-bukkit/src/main/java/com/fastasyncworldedit/bukkit/FaweBukkit.java:156]:
```suggestion
        plugins.sort(Comparator.comparing(Plugin::getName));
```

**Review by SirYwell — CHANGES_REQUESTED** [review]
Minor nitpick, looks good overall.

**Review by SirYwell — APPROVED** [review]

**Pull request merged**